skydriver/__main__.py

```diff
@@ -2,13 +2,31 @@
 
 import asyncio
 
-from . import server
-from .config import ENV, config_logging
+from . import database, k8s, server
+from .config import ENV, LOGGER, config_logging
 
 
 async def main() -> None:
-    """Start server."""
-    rs = await server.make()
+    """Establish connections and start components."""
+
+    # Mongo client
+    LOGGER.info("Setting up Mongo client...")
+    mongo_client = await database.create_mongodb_client()
+    LOGGER.info("Mongo client connected.")
+
+    # K8s client
+    LOGGER.info("Setting up k8s client...")
+    k8s_api = k8s.setup_k8s_client()
+    LOGGER.info("K8s client connected.")
+
+    # Scan Backlog Runner
+    LOGGER.info("Starting scan backlog runner...")
+    asyncio.create_task(k8s.scan_backlog.startup(k8s_api, mongo_client))
+    await asyncio.sleep(0)  # start up previous task
+
+    # REST Server
+    LOGGER.info("Setting up REST server...")
+    rs = await server.make(mongo_client, k8s_api)
```

Comment:
Nitpick: the order of the backlog arguments are opposite of the server arguments.  Maybe just pass by keyword args so you can order them however you like here?

Reply:
I swapped the order [b9d1be6](https://github.com/WIPACrepo/SkyDriver/pull/36/commits/b9d1be63dcb00e398e27476165c74994856df49c)